test.c

```diff
@@ -0,0 +1,99 @@
+#include "wslib.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+void main(int argc, char *argv[]) {
+
+  /*
+  char tmp_name[200];
+  strcpy(tmp_name,"styl.css");
+
+
+  struct safeObject myObject;
+
+  createSafeObjectFromFile(&myObject,tmp_name);
+
+  char *ptr=myObject.sObjectStart;
+  for(int i=0;i<myObject.sSize;i++){
+  printf("%c",*ptr);ptr++;
+  }
+
+  ptr=myObject.sObjectStart;
+  printf("\n----------------\n");
+  for(int i=0;i<myObject.sSize/2;i++){
+  printf("%c",getSafeCharCode(ptr));
+  ptr+=2;}
+
+
+  free(ptr);
+  */
+
+  if (argc == 2) {
+    if (strcmp(argv[1], "-h") == 0) {
+      printf("-h this help\nto output-file source-file\nfrom source-file  "
+             "output-file\nexample:\nWSsafeChars.exe to output.txt myzip.zip  "
+             "--- copies  ");
+      printf("from myzip.zip to  output.txt in safeChars code\nWSsafeChars.exe "
+             "from output.txt mp.zip  --- copies from output.txt to mp.zip");
+    }
+  }
+  if (argc == 4) {
+
+    if (strcmp(argv[1], "to") == 0) {
+      struct safeObject myObject;
+      createSafeObjectFromFile(&myObject, argv[3]);
+      FILE *out = fopen(argv[2], "wt");

```

Comment:
```suggestion
      FILE *out = fopen(argv[2], "w");

```
`t` jest domyślne

https://stackoverflow.com/a/23051095/384309